_posts/2018-11-29-blog-build-chrome-ringcentral-widgets-extension-for-crms.markdown

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title:  "Build Chrome RingCentral widgets extension for CRMs"
+date:   2018-11-29 13:31:07 +0800
+categories: blog
+---
+
+## Why is it need?
+
+- First, and the important reason, I guess, is Our clients need it, so for big CRM, like salesforce, we even build special app for it. For not that big CRMs, still have clients need it too, we may not build a special app for it. But we have the powerful widgets, so we can build a chrome extension easily.
```

Comment:
If you are making a list, I'd suggest you drop the uses of "first" because you should be listing them in order already. Similarly, you don't have to say this is "the important reason" because yo should already be listing in order of importance. I'd avoid using "I guess" in a formal writing because it make you sound not as confident. 
Lastly, since this is in bullet point format, try to keep it short and to the point.